btr-web/btr-main-app/components/individual-person/summary-table/row/Index.vue

```diff
@@ -0,0 +1,140 @@
+<script setup lang="ts">
+import {
+  IndividualPersonSummaryTableRowActionButton,
+  IndividualPersonSummaryTableRowCessationEntry
+} from '#components'
+import { SiSchemaType } from '~/utils/si-schema/definitions'
+type ActionBtnT = {
+  action: Function,
+  actionArg?: number,
+  disabled: boolean,
+  icon?: string,
+  label: string
+}
+
+const props = defineProps<{
+  si: SiSchemaType,
+  index: number,
+  actionBtn?: ActionBtnT,
+  actionDropdownBtns?: ActionBtnT[]
+}>()
+
+const t = useNuxtApp().$i18n.t
+
+const rowBadges = computed(() => {
+  if (!props.si.ui.actions?.length) {
+    return
+  }
+  const badges = []
+  if (props.si.ui.actions?.includes(FilingActionE.ADD)) {
+    badges.push({ label: t('badges.new') })
+  }
+  if (props.si.ui.actions?.includes(FilingActionE.EDIT)) {
+    badges.push({ label: t('badges.updated') })
+  }
+  if (
+    props.si.ui.actions?.includes(FilingActionE.CEASE) ||
+    props.si.ui.actions?.includes(FilingActionE.HISTORICAL)
+  ) {
+    badges.push({ label: t('badges.ceased'), colour: 'gray' })
+  }
+  return badges
+})
+
+const opacityClass = computed(() => {
+  if (
+    props.si.ui.actions?.includes(FilingActionE.CEASE) ||
+    props.si.ui.actions?.includes(FilingActionE.HISTORICAL)
+  ) {
+    return 'opacity-55'
+  }
+  return ''
+})
+</script>
+
+<template>
+  <tr
+    class="border-t-[1px] border-gray-400"
+    data-cy="summary-table-row"
+  >
+    <td data-cy="summary-table-name">
+      <PersonInfoName
+        :badges="rowBadges"
+        :item="{
+          'legalName': si.name.fullName,
+          'alternateName': si.name.preferredName,
+          'birthDate': si.birthDate,
+          'class': opacityClass}"
+      />
+      <PersonInfoCitizenship
+        :class="opacityClass"
+        :nationalities="si.citizenships"
+      />
+    </td>
+    <td class="align-top" :class="opacityClass" data-cy="summary-table-details">
+      <PersonInfoDetails :item="si" />
+    </td>
+
+    <td :class="opacityClass" data-cy="summary-table-controls">
+      <PersonInfoControl :item="si" />
+    </td>
+
+    <td :class="opacityClass" data-cy="summary-table-dates">
+      <p v-for="date in si.effectiveDates" :key="date.startDate">
+        {{
+          $t('texts.dateRange', {
+            start: date.startDate ? date.startDate : $t('labels.unknown'),
+            end: date.endDate ? date.endDate : $t('labels.current')
+          })
+        }}
+      </p>
+    </td>
+    <!-- NOTE: opacityClass is not applied to the edit column on purpose -->
```

Comment:
Was it not working applying opacityClass to tr and then just resetting opacity for this td ?

Reply:
Not a deal breaker, no need to update it right now for the sake of time.

Reply:
I did not know there was a way to do what you are suggesting. I thought all children of a parent element with a certain opacity were stuck with it (unless they want less opacity)